domains/authorization/Authorization.API/API/Migrations/20240730104826_AddDefaultTerms.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace API.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddDefaultTerms : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.InsertData(
```

Comment:
Why not use on migration file?

Reply:
Fixed: https://github.com/Energinet-DataHub/energy-origin/pull/1310/commits/99abc05de6b469ca9431a6920ed1dffb024aa2b3